### PR TITLE
Customized the cursor and Hovering effect

### PR DESCRIPTION
**Pull request by Nih1tGupta**
FIXED ISSUE: #137 
Added gradient according to website requirement
https://github.com/Its-Aman-Yadav/Community-Site/assets/119615342/63e1bb04-928f-4910-a8dd-8b9c49ef6889

Please take a look and kindly accept my PR under GSSOC'24

### CODE REVIEW

**Comment by Nih1tGupta** [comment]
Also added hovering effects on testimonials cards.

File: css/style.css [hunk 1]
```diff
@@ -939,3 +954,15 @@ i {
...
+  height: 24px;
+  width: 24px;
+  border-radius: 24px;
+  background-color: black;
+  position: fixed; 
+  top: 0;
+  left: 0;
+  pointer-events: none;
+  z-index: 99999999;
+  border-radius: 10%;
```
Comment (Nih1tGupta) [inline css/style.css:967]: Please increase  border-radius (for circluar cursor)

**Review by Its-Aman-Yadav — CHANGES_REQUESTED** [review]
Please make it looks like a cursor not a circle

**Comment by Nih1tGupta** [comment]
Working  on it

**Pull request closed without merging**